paasta_tools/setup_tron_namespace.py

```diff
@@ -62,22 +75,49 @@ def main():
     else:
         logging.basicConfig(level=logging.WARNING)
 
-    try:
-        new_config = tron_tools.create_complete_config(
-            service=args.service,
-            soa_dir=args.soa_dir,
-        )
+        if args.all_namepsaces:
+            tron_config = tron_tools.load_tron_config()
+            try:
+                services = tron_tools.get_tron_namespaces_for_cluster(
+                    cluster=tron_config.get_tron_cluster())
+            except ConflictingNamespacesError as e:
+                log.error('Failed to list tron namespaces: {error}'.format(
+                    error=str(e)))
+                sys.exit(1)
+        else:
+            services = [args.service]
```

Comment:
Isn't `args.service` already a list because of `nargs`? Otherwise, what happens if multiple services are passed in?